amardeep28/task3/style.css

```
@@ -0,0 +1,157 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans|Permanent+Marker');
+body {
+  font-family: 'Open Sans', sans-serif;
+  background: linear-gradient(to bottom, #ece9e6, #ffffff);
+}
+#header {
```

You don't have to use `id` here. Directly use `header` selector since you have one one header element.
```suggestion
header {
```